sdk/extensions/tests.yml

```diff
@@ -0,0 +1,13 @@
+trigger: none
```

Comment:
We have pipeline generation tooling that should automatically pick-up and setup a pipeline for the ci.yml file but we currently still have to manually setup the tests pipeline. Work with @danieljurek if you need help setting that up.

Reply:
Done.